cmd/kcn/main.go

```diff
@@ -241,7 +80,7 @@ func init() {
 	app.Flags = append(app.Flags, nodecmd.DBMigrationFlags...)
 
 	cli.AppHelpTemplate = utils.GlobalAppHelpTemplate
-	cli.HelpPrinter = utils.NewHelpPrinter(cnHelpFlagGroups)
+	cli.HelpPrinter = utils.NewHelpPrinter(utils.CategorizeFlags(app.Flags))
```

Comment:
This is both suggestion and question. I think it is a good idea to apply this to kbn, too. kbn needs help print for flags and it would be great if kbn works like other binaries. However, even though `cli.HelpPrinter` is not defined in kbn, kbn has its own help print out. Do you know how the print works in kbn?

Reply:
As you said, we have other binaries such `kbn`, `homi` and `kgen`. They also have a help description for flags but don't have flag groups because they only have few flags.

Reply:
Oh, I understand now why you specified `kbn`. I found out that `kbn` has more flags than I expected. I think some flags are not necessary for `kbn`. Let's handle this later.